contracts/libraries/GnosisAuction.sol

```diff
@@ -118,6 +118,15 @@ library GnosisAuction {
         );
     }
 
+    function settleAuction(address gnosisEasyAuction, uint256 auctionID)
+        internal
+        returns (bytes32 clearingOrder)
+    {
+        clearingOrder = IGnosisAuction(gnosisEasyAuction).settleAuction(
+            auctionID
+        );
```

Comment:
Since this function is just a single line, could we just call it directly from the contract instead of moving it down to the library?

Reply:
Removed this from GnosisAuction library.
Directly calling the contract from Vaultlifecycle.